.github/workflows/deploy.yml

```
@@ -0,0 +1,22 @@
+name: deploy
+
+on:
+  pull_request: {}
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - uses: actions/checkout@v4
+    - uses: w3c/spec-prod@v2
+      with:
+        BUILD_FAIL_ON: warning
+        GH_PAGES_BRANCH: main
```

Yep, this does need to be a separate branch, especially for Respec, or it'll overwrite the spec source. Doing this will also speed up loading for people visiting the ED questionnaire.

```suggestion
        GH_PAGES_BRANCH: gh-pages
```